checkEligibility.js

```diff
@@ -4,90 +4,57 @@ const logger = require('./logger')
 const kms = require('./lib/kms-helper')
 const { SierraError, ParameterError } = require('./lib/errors')
 
-function patronCanPlaceTestHold (patronId, attempt = 1) {
+async function patronCanPlaceTestHold (patronId, attempt = 1) {
   logger.debug('Performing patronCanPlaceTestHold')
   const body = {
-    json: true,
-    method: 'POST',
-    body: {
-      recordType: 'i',
-      recordNumber: 10000000,
-      pickupLocation: 'maii2'
-    }
+    recordType: 'i',
+    recordNumber: 10000000,
+    pickupLocation: 'maii2'
+  }
+  // This try/catch block logic is inverted. The success case of this request results in an error, and the failure case results in a typical response.
+  let patronHoldsPossible
+  let response
+  try {
+    response = await wrapper.post(`patrons/${patronId}/holds/requests`, body)
+    logger.error('Error: Placing a test hold on a test item did not generate an error!')
+    return false
+  } catch (e) {
+    patronHoldsPossible = e.response.data.description === 'XCirc error : There is a problem with your library record.  Please see a librarian.'
```

Comment:
Why has this expectation changed? It was [originally checking for a different error](https://github.com/NYPL-discovery/patron-eligibility-service/blob/2e88294b688db7376470e78d3d0a3dff1e88b566/checkEligibility.js#L26). I believe that the "There is a problem with your library record" error checked above actually means that holds are _not_ possible on the card, so I believe this should be reverted to the original check.